src/lib/compile_time_info/gen.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
This is the ugliest part of this solution. We could potentially use https://github.com/stedolan/ppx_stage. But that lib is too old and we need to have it update to date. 

Right now I guess this hack is fine, as it's already used in `mina_version.normal`